### PR TITLE
Don't generate universal wheels

### PR DESCRIPTION
**Pull request by palango**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def read(*names, **kwargs):
 
 setup(
     name='raiden-services',
-    version='0.1.0',
```
Comment (LefterisJP) [inline setup.py]: Did the version go down here???

Comment (palango) [reply setup.py]: Yes, we never released before and I just wanted to reserve the PyPI package name

**Comment by ulope** [comment]
Why don’t you want universal wheels?

**Comment by ulope** [comment]
Answering my own question: Because universal implies Python 2 compatibility. TIL.

**Pull request merged**